### PR TITLE
Configurable stylus

### PR DESCRIPTION
**Pull request by Cactucs**
Use `_.assign` or `_.merge`? I used assign as in https://github.com/manGoweb/mango-cli/commit/7902ad08fe281dced17814ddd9a30f836526046f.

### CODE REVIEW

File: lib/tasks/styles.js [hunk 1]
```diff
@@ -25,13 +25,13 @@ module.exports = function(gulp, config) {
 
 		var task = gulp.src(config.styles, { base: config.src_folder })
 
-		var stylusOptions = {
+		var stylusOptions = require('lodash/object/assign')({}, {
```
Comment (enzy) [inline lib/tasks/styles.js]: do not place require calls inline

Comment (Cactucs) [inline lib/tasks/styles.js]: Why not?

Comment (enzy) [inline lib/tasks/styles.js]: To respect a code organization style around

Comment (Cactucs) [inline lib/tasks/styles.js]: :ok: :white_check_mark: Done

File: lib/tasks/styles.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ module.exports = function(gulp, config) {
 		var sass = require('gulp-sass')
 		var sourcemaps = require('gulp-sourcemaps')
 		var stylus = require('gulp-stylus')
+		var lodash = require('lodash')
```
Comment (enzy) [inline lib/tasks/styles.js]: you can do even better by requiring only the assign function

Comment (Cactucs) [inline lib/tasks/styles.js]: :white_check_mark:

**Comment by enzy** [comment]
:+1:

**Pull request merged**